docs/sources/shared/stability/community.md

```diff
@@ -0,0 +1,8 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/shared/stability/community/
+description: Shared content, community contributed
+headless: true
+---
+
+> **Community**: This component is developed, maintained, and supported by the Alloy user community.
+> Grafana doesn't offer commercial support for this component.
```

Comment:
Could you add that to use it the user must set the flag "--feature.community-components.enabled" to true with the run command?